packages/office-ui-fabric-react/src/components/Rating/Rating.tsx

```diff
@@ -135,15 +130,11 @@ export class Rating extends BaseComponent<IRatingProps, IRatingState> {
         id={ `${this._labelId}-${rating}` }
         className={ css('ms-Rating-labelText', styles.labelText) }
       >
-        { `${rating} ${text}` }
+        { `${rating} of ${this.props.max} ${text} selected` }
```

Comment:
this is not localizable, if you want to have that string you have to pass it in

Reply:
Yes, everyone has to pass it in. And we localize everything - when we localize. Look at the date picker for an example

Reply:
changed it to aria-labelby and passed from parent

Reply:
I'm confused; i still see a hardcoded string here.

Reply:
Still seeing English strings...

Reply:
Plus this is making all sorts of assumptions about the order of things in the string. The *whole string* needs to be passed in.

Reply:
Ack. Updating with fix.